dataanalysis/core.py

```diff
@@ -1,6 +1,6 @@
 from io import StringIO
 from pathlib import Path
-import collections
+import collections.abc
```

Comment:
why leave import?

Reply:
Since the call for Iterable has changed, I don't think it is needed. I will remove it.